addons/mail/static/src/component_hooks/use_drag_visible_dropzone/use_drag_visible_dropzone.js

```diff
@@ -75,9 +75,7 @@ function useDragVisibleDropZone() {
      */
     function _onDropListener(ev) {
         dragCount = 0;
-        if (dropzoneRef.comp && !dropzoneRef.comp.contains(ev.target)) {
```

Comment:
@alexkuhn Do you remember why this condition was added ?

Reply:
Don't remember, so ok to remove.